civic.json

```diff
@@ -0,0 +1,8 @@
+{
+    "status": "Beta", 
```

Comment:
Why is the status `Beta`? This app has been deployed to production for years at http://boston.adoptahydrant.com/